### ISSUE TITLE
Relative Diff of BERT(optimize on FP16 vs FP32) is high, when using  polygraphy api on TRT8.6.1

### ISSUE THREAD

**Issue by b4b4o** [NVIDIA/TensorRT#3486]
## Description

when I use polygraphy python api on TRT8.6.1 to optimize BERT onnx model, which download from Hugging Face, the mean of Relative Difference result on (FP16/FP32) is high(about 1%).

## Log
[I]             Relative Difference | Stats: mean=0.011178, std-dev=0.13094, var=0.017146, median=0.0041003, min=7.3937e-08 at (0, 0, 22953), max=15.88 at (0, 0, 2853), avg-magnitude=0.011178
[I]                 ---- Histogram ----
                    Bin Range        |  Num Elems | Visualization
                    (7.39e-08, 1.59) |      30508 | ########################################
                    (1.59    , 3.18) |          6 |
                    (3.18    , 4.76) |          3 |
                    (4.76    , 6.35) |          2 |
                    (6.35    , 7.94) |          2 |
                    (7.94    , 9.53) |          0 |
                    (9.53    , 11.1) |          0 |
                    (11.1    , 12.7) |          0 |
                    (12.7    , 14.3) |          0 |
                    (14.3    , 15.9) |          1 |
[E]         FAILED | Output: 'logits' | Difference exceeds tolerance (rel=1e-05, abs=1e-05)
[E]     FAILED | Mismatched outputs: ['logits']
[E] Accuracy Summary | trt-runner-N0-11/24/23-15:22:50 vs. trt-runner-N1-11/24/23-15:22:50 | Passed: 0/1 iterations | Pass Rate: 0.0%

## Environment

**TensorRT Version**: 8.6.1
**NVIDIA Driver Version**: 470.82
**CUDA Version**: 11.4
**Python Version **: 3.9.17
**polygraphy Version**: 0.47.1
**onnx Version**: 1.12.0

onnx model path : https://huggingface.co/bert-base-uncased/blob/main/model.onnx

## Code
```
from polygraphy.backend.trt import EngineFromNetwork, NetworkFromOnnxPath, TrtRunner, CreateConfig
from polygraphy.comparator import Comparator, CompareFunc


network = NetworkFromOnnxPath("bert.onnx")

config = CreateConfig(fp16=True)
engine_allfp16 = EngineFromNetwork(network, config)
config2 = CreateConfig()
engine_allfp32 = EngineFromNetwork(network, config2)

runners_1 = [
        TrtRunner(engine_allfp32),
        TrtRunner(engine_allfp16)
]

run_results = Comparator.run(runners_1)
Comparator.compare_accuracy(run_results, compare_func=CompareFunc.simple(atol=1e-5))
```

**Comment by zerollzeng** [comment]
When you build FP16 engine for BERT, you should see some warnings out FP16 underlfow/overflow. Especially for LayerNorm. e.g. 
```
[11/25/2023-07:02:44] [W] [TRT] Running layernorm after self-attention in FP16 may cause overflow. Exporting the model to the latest available ONNX opset (later than opset 17) to use the INormalizationLayer, or forcing layernorm layers to run in FP32 precision can help with preserving accuracy.
```
Could you please try fallback the LayerNorm(and other layers that throw warning) to FP32 and try again?

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, thanks all!
